apps/server/migrations/0010_session.ts

```diff
@@ -15,6 +16,7 @@ export async function up(db: Kysely<unknown>): Promise<void> {
     .addColumn("station_id", "text", (col) =>
       col.notNull().references("station.id").onDelete("cascade"),
     )
+    .addColumn("duration_ms", "integer", (col) => col.notNull())
```

Comment:
could you add the schema change to a separate file and use `alterTable`. This way we no longer drop the tables.